.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+
+{
+  "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+  },
+  "extends": [
+      "plugin:react/recommended",
+      "airbnb"
+  ],
+  "parserOptions": {
+      "ecmaFeatures": {
+          "jsx": true
+      },
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+  },
+  "plugins": [
+      "react"
+  ],
+  "rules": {
+      "react/jsx-filename-extension": [
+          "warn", { "extensions": [".js", ".jsx"] }
+      ]
+  }
+}
```

Comment:
Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/react-redux).

![image](https://user-images.githubusercontent.com/36057474/71924065-8b46a000-318e-11ea-8f37-02c188bde393.png)

See https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

Reply:
I did not push my last changes and this files lacks:

```
      "import/no-unresolved": "off",
      "no-shadow": "off"
```

Thanks